scripts/generate-sw.js

```diff
@@ -20,4 +30,4 @@ fs.writeFileSync(
       /%CSS_ASSETS/gi,
       `'/assets/css/vendor/foundation.min.css?v=6.2.4', '/assets/css/bundle.css'`
     )
-);
+);*/
```

Comment:
Instead have `sourceFile` variable and let the code be the way it was, only change the value of `sourceFile`

Reply:
Fixed.